packages/terra-theme-properties/src/scss/semantic-names.scss

```diff
@@ -323,13 +323,14 @@
 
 // =======================================
 // terra-modal-manager
-// https://github.com/cerner/terra-core/tree/main/packages/terra-modal-manager
+// https://github.com/cerner/terra-framework/tree/main/packages/terra-modal-manager
 // https://jira2.cerner.com/browse/UXPLATFORM-10006
 // =======================================
 .redwood-theme {
   --modal-background-color: var(--container-style-1-background-color);
   --modal-box-shadow: var(--rds-box-shadow-md);
   --modal-border-radius: var(--rds-border-radius-md);
+  // TODO add --modal-foreground-color variable or remove this line is inherit color works
```

Comment:
I still think we don't want a UX theme variable here.

Reply:
Removed here: https://github.com/cerner/terra-framework/pull/1994/commits/dcf6be070dc631b1296ea771e9b984a402cd7616